src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ClusterBox.scss

```diff
@@ -0,0 +1,53 @@
+@import '@influxdata/clockface/dist/variables.scss';
+
+.cluster-box.hoverable {
+  border: 2px solid $cf-grey-25;
+  max-width: none;
+
+  &:hover {
+    border-color: $c-laser;
+    > .cluster-box-title {
+      background-color: $c-laser;
+    }
+  }
+
+  > .cluster-box-logo {
+    justify-content: center;
+  }
+}
+
+.cluster-box {
+  display: flex;
+  flex: 1;
+  margin: 5px;
+  border-radius: 5px;
+  cursor: pointer;
+  max-width: 350px;
+
+  &.selected {
+    border-color: $c-laser;
+    > .cluster-box-title {
+      background-color: $c-laser;
+    }
+  }
+
```

Comment:
Do you think this `SelectableCard` component will take care of all this custom styling? 

https://influxdata.github.io/clockface/?path=/story/components-cards-selectablecard--selectablecard

Reply:
didn't know about this component. will update it! thank you!